README.md

```diff
@@ -1,27 +1,50 @@
 # Callisto Balance Calculator Service
 
-## Building the project
+Callisto Balance Calculator is a part of Callisto project.
+The best way to run the service is to leverage the [callisto-localdev solution](https://github.com/UKHomeOffice/callisto-localdev).
+
+## Running as part of the LocalDev environment
+
+1. Clone LocalDev repository from https://github.com/UKHomeOffice/callisto-localdev and run it locally as described in [Scenario 1](https://github.com/UKHomeOffice/callisto-localdev#scenario-1-running-callisto-without-need-to-edit-code-base-eg-demo-purposes).
+
+2. From the LocalDev project root, stop the service by running `docker compose stop callisto-balance-calculator` command.
+
+3. Ensure you have configured your [Authentication for GitHub Packages](#authentication-for-github-packages)
+
+4. Clone the service repository, and from its root directory run command `docker compose up -d`
+
+After successful start, you should be able to work with the service code, and see all changes without restaring the docker container.
+
+## Authentication for Github Packages
 
-### Github Package dependencies
 In order to pull in Github package dependencies you will need a Github Personal Access Token.
 This token will need the minimum of 'packages:read' permissions.
 
-Update your .m2/settings.xml file to contain the <servers><server> tags like timecard_settings.xml
-The token will need to live within your local .m2/settings.xml file as the password
+This can be configured in your .m2/settings.xml file.
+
+E.g.
+
+```xml
+<servers>
+  <server>
+      <id>github-packages</id>
+      <username>[Insert GitHub username here]</username>
+      <password>{[Insert GitHub Personal Access Token here]}</password>
+  </server>
+</servers>
+```
 
 For more info see:
 [https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry)
 
 Then run the following to build the project
 
-```sh
-$ mvn clean install
-```
+## Debugging
+
+Assuming you know how to debug java service, the principal is the same when using [callisto-localdev solution](https://github.com/UKHomeOffice/callisto-localdev). When running using docker compose, the debugger is exposed on the default port (50211). In the case of the debugger port conflicts, change both the ports and the entrypoint section in the docker-compose.yml
+
+## Kafka
 
-## Actuator Kafka Error Endpoint
-We have a counter configured for actuator which will give a metric on how many times deserialization has failed for kafka event messages.  
-This can be accessed at
+Project requires a Kafka instance with all required permissions to write to topic. If [running the service within LocalDev solution](#running-as-part-of-the-localdev-environment), no futher configuration is required.
```

Comment:
This service doesn't write to the topic it reads or consumes

Reply:
Fixed